setup.py

```diff
@@ -45,7 +45,7 @@ def get_long_description():
     "sbert": ["sentence_transformers"],
     "doc2vec": ["gensim"],
     "tensorflow": ["tensorflow~=2.0"],
-    "dev": ["check-manifest"],
+    "dev": ["black", "check-manifest", "flake8", "flake8-isort", "isort"],
     "test": ["coverage", "pytest"],
 }
 DEPS["all"] = DEPS["sbert"] + DEPS["doc2vec"] + DEPS["dev"]
```

Comment:
I think it is better to remove `DEPS["dev"]` from `all` now.

Reply:
updated as requested